compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MainMethodGen.java

```diff
@@ -69,6 +69,9 @@
  */
 public class MainMethodGen {
 
+    public static final String INIT_FUTURE_VAR = "initFutureVar";
```

Comment:
Why do we have `Var` suffix in only initFuture variable? Shall we make all three names consistent as they are for similar purpose?